horde/apis/models/v2.py

```diff
@@ -1278,6 +1279,232 @@ def __init__(self, api):
                 ),
             },
         )
+        self.response_model_known_model_md = api.model(

```

Comment:
There's a separate file for image and text worker models. kobold_v2.py and stable_v2.py. You need to store the type-specific models in the right file to keep things somewhat organized. 

Common data in both types should stay in v2.py and then use model inheritance to extend for each type.